### PR TITLE
Refactor/extract stubs from testhelper

### PR DESCRIPTION
**Pull request by FarouqJalabi**
Extracted stubs from test_helper.rb. It was starting to get to big.

### CODE REVIEW

File: test/meeting_test.rb [hunk 1]
```diff
@@ -8,6 +8,22 @@ def setup
     create_and_stub_client
   end
 
+  # rubocop:disable Metrics/LineLength, Metrics/ MethodLength
```
Comment (mrtnin) [inline test/meeting_test.rb:11]: Lets disable these rules for the test/** pattern

**Review by mrtnin — APPROVED** [review]
I agree this needs to be extracted. However I think there's potential for a better dsl and less verbose setup. Wdyt?

**Comment by FarouqJalabi** [comment]
> I agree this needs to be extracted. However I think there's potential for a better dsl and less verbose setup. Wdyt?

I think so to but couldn't really come up with anything smart. Do you have any ideas?

**Comment by mrtnin** [comment]
> > I agree this needs to be extracted. However I think there's potential for a better dsl and less verbose setup. Wdyt?
> 
> I think so to but couldn't really come up with anything smart. Do you have any ideas?

No, not at the moment. Let's revisit this later.

**Pull request closed without merging**